mlflow/server/js/src/experiment-tracking/components/CompareRunView.tsx

```diff
@@ -43,109 +37,116 @@ type CompareRunViewProps = {
   intl: IntlShape;
 };
 
-type CompareRunViewState = any;
-export class CompareRunView extends Component<CompareRunViewProps, CompareRunViewState> {
-  compareRunViewRef: any;
-  runDetailsTableRef: any;
-
-  constructor(props: CompareRunViewProps) {
-    super(props);
-    this.state = {
-      tableWidth: null,
-      onlyShowParamDiff: false,
-      onlyShowTagDiff: false,
-      onlyShowMetricDiff: false,
-    };
-    this.onResizeHandler = this.onResizeHandler.bind(this);
-    this.onCompareRunTableScrollHandler = this.onCompareRunTableScrollHandler.bind(this);
-
-    this.runDetailsTableRef = React.createRef();
-    this.compareRunViewRef = React.createRef();
-  }
-
-  onResizeHandler(e: any) {
-    const table = this.runDetailsTableRef.current;
-    if (table !== null) {
-      const containerWidth = table.clientWidth;
-      this.setState({ tableWidth: containerWidth });
-    }
-  }
+const CompareRunView: React.FC<CompareRunViewProps> = (props) => {
+  const {
+    runInfos,
+    experimentIds,
+    metricLists,
+    paramLists,
+    tagLists,
+    runNames,
+    runDisplayNames,
+    intl,
+    runUuids,
+    hasComparedExperimentsBefore,
+  } = props;
```

Comment:
nit: can be destructured directly in props

```suggestion
const CompareRunView: React.FC<CompareRunViewProps> = ({
    runInfos,
    experimentIds,
    metricLists,
    paramLists,
    tagLists,
    runNames,
    runDisplayNames,
    intl,
    runUuids,
    hasComparedExperimentsBefore,
  }) => {
```